en/docs/reference/mediators/data-mapper-mediator.md

```diff
@@ -174,6 +174,17 @@ The operations that the Data Mapper supports as shown below.
 
 - **match** – check whether the input match with a (JS) Regular Expression
 
+## AI Data Mapper
+The MI Copilot AI Assistant will allow you to seamlessly generate the input-output mapping. Simply load the input and output schema (of any type) to the relevant sections as shown below and click **Map**. A **Clear** button is also available to clear all mappings. 
+
+!!! info "WSO2 Account and Usage Limits"
+    To use this feature, you must have a WSO2 account. If you do not have one, attempting to access Copilot or the mapping feature will prompt you to create an account. If your account exceeds the usage limit, it will be refreshed after the specified time limit.
+
+!!! info "Review and Fine-tune"
+    Since AI powers the automated mapping, it may contain errors. Always verify the accuracy of the generated mappings using the `</>` button to access the TypeScript (TS) file and fine-tune them.
```

Comment:
```suggestion
    Since AI powers the automated mapping, it may contain errors. Always verify the accuracy of the generated mappings. To fine tune the mappings you can use the `</>` button and edit the TypeScript (TS) file.
```